File: Numbers/PrimeFactors.java
public class PrimeFactors {
    public static boolean PrimeNumbers(int n){
        int count=0;
        for(int i=1;i<=n;i++){
            if(n%i==0){
                count++;
            }
        }
        if(count==2){
            return true;
        }else{
            return false;
        }

    }
    public static void main(String[] args) {
        int num=60;
        for(int i=1;i<=num;i++){
            if(num%i==0 && PrimeNumbers(i)){
                System.out.println(i+":is a prime factor");
            }
        }

    }
}

File: Problem Solving Python/Numbers/PrimeFactors.py
def Prime(n):
    count=0
    for i in range(1,n+1):
        if(n%i==0):
            count+=1
    if(count==2):
        return True
    else:
        return False

if __name__=='__main__':
    n=int(input("Enter a number: "))
    for i in range(1,n+1):
        if(n%i==0 and Prime(i)):
            print(i," : is a prime factor")